source/features/link-to-gihub-io.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+import LinkExternalIcon from 'octicon/link-external.svg';
+
+import features from '.';
+import {getRepositoryInfo} from '../github-helpers';
+
+function repoListInit(): void {
+	for (const repo of select.all<HTMLAnchorElement>('a[href$=".github.io"][itemprop="name codeRepository"]')) {
+		repo.after(
+			' ',
+			<a
+				href={`https://${repo.textContent!.trim()}`}
+				target="_blank"
+				rel="noopener noreferrer"
+			>
+				<LinkExternalIcon className="v-align-middle"/>
+			</a>
+		);
+	}
+}
+
+async function repoInit(): Promise<void> {
+	const repoTitle = await elementReady('[itemprop="name"]')!;
+	repoTitle!.after(
+		<a
+			href={`https://${getRepositoryInfo().name!}`}
+			target="_blank"
+			rel="noopener noreferrer"
+		>
+			<LinkExternalIcon className="v-align-middle"/>
+		</a>
+	);
+}
+
+void features.add({
+	id: __filebasename,
+	description: 'Add a link to visit the user’s github.io website from its repo.',
+	screenshot: 'https://user-images.githubusercontent.com/31387795/94045261-dbcd5e80-fdec-11ea-83fa-30bb673cc26e.jpg'
+}, {
+	include: [
+		pageDetect.isUserProfileRepoTab
+	],
+	init: repoListInit
+}, {
+	include: [
+		pageDetect.isRepo
+	],
+	exclude: [
+		() => !getRepositoryInfo()!.name!.endsWith('.github.io')
+	],
+	init: repoInit
+});
```

Comment:
```suggestion
}, {
	include: [
		pageDetect.isRepo
	],
	exclude: [
		() => !getRepositoryInfo()!.name!.endsWith('.github.io')
	],
	init: initRepo
}, {
	include: [
		pageDetect.isUserProfileRepoTab
	],
	init() {
		observeElement('#user-repositories-list', initRepoList);
		return false;
	}
});
```
Note: I made a few changes here.

First I changed the order since the Repo one is the one on top of the features.add so that goes first.

I also changed the the word `init` goes before the function name. (Both functions)

The last change is that we need to observe the search bar for any changes. Since once a user makes a search the feature will disappear. 
Don't forget to add the ObserveElement to the imports.